atlascli.md

```diff
@@ -0,0 +1,66 @@
+# AtlasCLI (Pre-Release)
```

Comment:
The product name is "Atlas CLI", not sure if we should add a space here and all the AtlasCLI below.

Reply:
updated